### PR TITLE
[BULK] DocuTune - Rebranding

### PR DESCRIPTION
**Pull request by alexbuckgit**
**Global effort for the Microsoft Learn platform rebrand**

The Microsoft team is applying changes to content for the Microsoft Learn platform rebrand. Changes include modifications to current Microsoft Learn and Microsoft Docs terminology as well as associated domain name and content URL path changes.

Please review this draft PR and provide any necessary comments as soon as possible. Because these fixes are tightly scoped, we have approval to automatically merge these changes to the default branch for publication.

This PR was generated via DocuTune. It includes only targeted fixes and minor formatting adjustments.

CorrelationId: d8c4c3ac-d49a-4c95-9ddc-9822ca297907
InitiativeId: docutune-rebranding-2022-09
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

**Comment by damabe** [comment]
#docutune-ready

**Comment by jehchow** [comment]
Please do not merge until our team has a chance to review.

File: .github/ISSUE_TEMPLATE/general-issue-template.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ assignees: carlyrevier, jehchow
 
 Welcome to the Microsoft Docs contributor guide feedback form!
 
-Issues submitted here are reviewed by the repo managers of the Docs contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://docs.microsoft.com/en-us/contribute/?branch=main) only. Requests for updates to products or product documentation will be closed.
+Issues submitted here are reviewed by the repo managers of the Docs contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://review.learn.microsoft.com/contribute/?branch=main) only. Requests for updates to products or product documentation will be closed.
```
Comment (jehchow) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:15]:
```suggestion
Issues submitted here are reviewed by the repo managers of the Docs contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://learn.microsoft.com/contribute) only. Requests for updates to products or product documentation will be closed.
```

File: .github/ISSUE_TEMPLATE/general-issue-template.md [hunk 2]
```diff
@@ -12,12 +12,12 @@ assignees: carlyrevier, jehchow
...
 
 If you have feedback about a product or product documentation, use one of these outlets instead:
 
 - For Microsoft Learn documentation: The feedback area at the bottom of an article. If there is none, post to our [Feedback Portal](https://feedbackportal.microsoft.com/feedback/).
 
-- For Microsoft Learn training content: The feedback link at the bottom of each unit. If there is none, use the **Report feedback** tool at the very bottom of [this article](https://review.docs.microsoft.com/en-us/learn/support/troubleshooting?branch=main#report-feedback).
+- For Microsoft Learn training content: The feedback link at the bottom of each unit. If there is none, use the **Report feedback** tool at the very bottom of [this article](https://review.learn.microsoft.com/learn/support/troubleshooting?branch=main#report-feedback).
```
Comment (jehchow) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:21]:
```suggestion
- For Microsoft Learn training content: The feedback link at the bottom of each unit. If there is none, use the **Report feedback** tool at the very bottom of [this article](https://learn.microsoft.com/learn/support/troubleshooting?branch=main#report-feedback).
```
Thanks for catching this one.

**Review by jehchow — COMMENTED** [review]
Just one suggested change.

**Comment by alexbuckgit** [comment]
Hi @jehchow -

Please merge, thanks!

#docutune-request-approval

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Contributor guide pull request
 
-Thanks for contributing to the [docs.microsoft.com contributor guide](https://docs.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.
+Thanks for contributing to the [Microsoft Docs contributor guide](https://learn.microsoft.com/contribute/?branch=main). Please read these instructions for processing your pull request (PR). Once you've read this text, delete it and write a brief description of the changes you're proposing.
```
Comment (jennsteiner74) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Update to: Microsoft documentation contributor guide

File: Contribute/index.md [hunk 4]
```diff
@@ -11,7 +11,7 @@ ms.custom: external-contributor-guide
 
 # Microsoft Docs contributor guide overview
```
Comment (jennsteiner74) [inline Contribute/index.md:12]: Should this be Microsoft Learn contributor guide?

File: Contribute/index.md [hunk 5]
```diff
@@ -11,7 +11,7 @@ ms.custom: external-contributor-guide
...
 
-Welcome to the [Microsoft Docs](https://learn.microsoft.com) contributor guide!
+Welcome to the Microsoft Docs contributor guide!
```
Comment (jennsteiner74) [inline Contribute/index.md:14]: Should this be Microsoft Learn contributor guide?

File: .github/ISSUE_TEMPLATE/general-issue-template.md [hunk 6]
```diff
@@ -12,12 +12,12 @@ assignees: carlyrevier, jehchow
 
 Welcome to the Microsoft Docs contributor guide feedback form!
```
Comment (jennsteiner74) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:13]: Microsoft Learn contributor guide

File: .github/ISSUE_TEMPLATE/general-issue-template.md [hunk 7]
```diff
@@ -12,12 +12,12 @@ assignees: carlyrevier, jehchow
...
 
-Issues submitted here are reviewed by the repo managers of the Docs contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://docs.microsoft.com/en-us/contribute/?branch=main) only. Requests for updates to products or product documentation will be closed.
+Issues submitted here are reviewed by the repo managers of the Docs contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://learn.microsoft.com/contribute) only. Requests for updates to products or product documentation will be closed.
```
Comment (jennsteiner74) [inline .github/ISSUE_TEMPLATE/general-issue-template.md:15]:
```suggestion
Issues submitted here are reviewed by the repo managers of the Learn contributor guide, **not product teams.** Your issue should pertain to the documentation in this [contributor guide](https://learn.microsoft.com/contribute) only. Requests for updates to products or product documentation will be closed.
```

File: Contribute/how-to-write-links.md [hunk 8]
```diff
@@ -10,9 +10,9 @@ ms.date: 03/31/2020
...
+This article describes how to use hyperlinks from pages hosted at Microsoft Learn. Links are easy to add into markdown with a few varying conventions. Links point users to content in the same page, in other neighboring pages, or on external sites and URLs.
 
-The Microsoft Docs backend uses Open Publishing Services (OPS), which supports [CommonMark][]-compliant markdown parsed through the [Markdig][] parsing engine. This markdown flavor is mostly compatible with [GitHub Flavored Markdown (GFM)][GFM], as most docs are stored in GitHub and can be edited there. Additional functionality is added through Markdown extensions.
+The Microsoft Learn backend uses Open Publishing Services (OPS), which supports [CommonMark][]-compliant markdown parsed through the [Markdig][] parsing engine. This markdown flavor is mostly compatible with [GitHub Flavored Markdown (GFM)][GFM], as most docs are stored in GitHub and can be edited there. Additional functionality is added through Markdown extensions.
```
Comment (jennsteiner74) [inline Contribute/how-to-write-links.md:15]: Do we want to leave "docs" as is or change to documents?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (jehchow) [reply .github/PULL_REQUEST_TEMPLATE.md:3]: At present time, the contributor guide title will stay the same.

File: Contribute/index.md (hunk 5 above)
Comment (jehchow) [reply Contribute/index.md:14]: At present time, the contributor guide title will stay the same.

File: .github/ISSUE_TEMPLATE/general-issue-template.md (hunk 6 above)
Comment (jehchow) [reply .github/ISSUE_TEMPLATE/general-issue-template.md:13]: At present time, the contributor guide title will stay the same.

File: Contribute/how-to-write-links.md (hunk 8 above)
Comment (jehchow) [reply Contribute/how-to-write-links.md:15]: This can stay as "docs".

**Review by jehchow — APPROVED** [review]

**Pull request merged**